web/src/css/custom.css

```diff
@@ -116,21 +116,18 @@
   opacity: 0.7;
 }
 
-.theme-doc-sidebar-item-category-level-1 {
-  margin-bottom: var(--sidebar-category-spacing);
+.theme-doc-sidebar-item-category-level-1,.theme-doc-sidebar-item-link-level-1 {
+  margin-bottom: var(--sidebar-item-level-1-spacing);
 }
 
-.theme-doc-sidebar-item-category-level-1 > .menu__list {
-  /* remove indentation from 1st level categories */
-  padding-left: 0;
+.theme-doc-sidebar-item-category-level-1 > .menu__list-item-collapsible > .menu__link,
+.theme-doc-sidebar-item-link-level-1 > .menu__link {
+  color: var(--sidebar-item-level-1-color);
+  font-weight: bold;
 }
 
-.theme-doc-sidebar-item-category-level-1
-  > .menu__list-item-collapsible
-  > .menu__link {
-  color: var(--sidebar-category-color);
-  font-weight: bold;
-  font-size: var(--ifm-h6-font-size);
+.theme-doc-sidebar-item-category > .menu__list-item-collapsible > .menu__link {
```

Comment:
```suggestion
.theme-doc-sidebar-item-category-level-1 > .menu__list-item-collapsible > .menu__link {
```
Might I suggest this change, it feels cleaner since only the top level is yellow.

| ![Screenshot 2024-05-28 at 10 56 36](https://github.com/wasp-lang/wasp/assets/2223680/22a7ef71-6796-42d9-8df2-9d62bf93cef0) |  ![Screenshot 2024-05-28 at 11 00 38](https://github.com/wasp-lang/wasp/assets/2223680/79fa078f-7dc9-4356-8088-d8fcbe8be4bc) |
| --- | --- |
| Before | After |

Reply:
But I did it on purpose, to make it clear what is "openable". Otherwise, once you open those Operations, it is a "see" of white/black, it is hard to see who is parent a bit. Notice only top level is bold though.

Reply:
Ok, done! I don't want us to shy that much from design though, we can try, but your are right if docusaurus has it that way, probably be should also.